Sources/ProjectDescription/ResourceSynthesizer.swift

```diff
@@ -85,7 +85,7 @@ public struct ResourceSynthesizer: Codable, Equatable, Sendable { // swiftlint:d
             templateType: templateType,
             parser: .strings,
             parserOptions: parserOptions,
-            extensions: ["strings", "stringsdict"]
+            extensions: ["strings", "stringsdict", "xcstrings"]
```

Comment:
Changes introduced in [this PR](https://github.com/tuist/SwiftGen/pull/1) extend Strings parser, that's why there no need to have another parser anymore

Reply:
Thanks to that ☝️ there's also no need to have [stringsCatalog](https://github.com/tuist/XcodeGraph/blob/1c7ef35380a4c5d316f3fda68b6dd8140ab0f287/Sources/XcodeGraph/Models/ResourceSynthesizer.swift#L18) case anymore